packages/react/src/analytics/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./analytics";
```

Comment:
We should probably move away from the `folder-name/index.ts` naming convention. I thought it had its advantages but I far overstimated them. It's probably better to avoid the folder/index structure and name files using kebab case. In case a module needs to be split up into multiple submodules it still makes sense to have a folder with an index file, but this doesn't seem to be the case.

Reply:
Fixed in 4b248304b183e32434a4dcc4198173a3cfe055a3.

Reply:
On this I wrote that it makes sense if we have a module with multiple submodules needed for clarity. It seems like this is the case. In this scenario it made sense to have a folder with the index exporting the 2 submodules, sorry.

Reply:
Fixed in 6490196e9b0b770b39a522af6265e9917ec2a327.